src/main/java/com/customer/controller/CreditCardController.java

```diff
@@ -0,0 +1,48 @@
+package com.customer.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.customer.exception.ForbiddenException;
+import com.customer.exception.InvalidRequest;
+import com.customer.model.CreditCard;
+import com.customer.service.CreditCardServiceImpl;
+
+@RestController
+@RequestMapping("/api")
+public class CreditCardController {
+	
+	private CreditCardServiceImpl creditCardServiceImpl;
+	
+	// Constructor autowiring
+	@Autowired
+	public CreditCardController(CreditCardServiceImpl creditCardServiceImpl) {
+		this.creditCardServiceImpl = creditCardServiceImpl;
+	}
+	
+	/* Post request */
+	@PostMapping("/creditCard")
+	public ResponseEntity<CreditCard> createCreditCardInfo(@RequestBody CreditCard creditCardInfo) throws InvalidRequest {
+		if (creditCardInfo != null) 
+			return ResponseEntity.ok().body(creditCardServiceImpl.createCreditCardInfo(creditCardInfo));
```

Comment:
why do u create the body directly... what if `creditCardServiceImpl.createCreditCardInfo(creditCardInfo)` throws an error.. what would happen ?

Reply:
It will return custom error message .